### ISSUE TITLE
[FB events] unable to save connector with default value

### ISSUE THREAD

**Issue by nicofrand** [cozy-labs/konnectors#738]
When the user does not want to modfiy the default value of the calendar's name input, he can't click on the save button, which is disabled.

See https://forum.cozy.io/t/mes-comptes/4201

**Comment by jacquarg** [comment]
Yes, sorry, we have something similar with the Orange Mobile connector : you have to check the opt-in to activate the save button.

On the "code" side, it's because the underlying statefull form library expect that the form is dirty to allows the click on **save** . But in these cases, the only field is optional, and so the UX is unacurate ...
